db/schema.rb

```diff
@@ -92,6 +92,7 @@
     t.datetime "created_at", precision: 6, null: false
     t.datetime "updated_at", precision: 6, null: false
     t.string "alt_text", default: "", null: false
+    t.string "page_name"
```

Comment:
Rather than linking an asset to a ContentPage by it's title, can add a `content_page_id` reference - then have `has_one :content_page` (or many if we decide to go that way, in which case a join table would be needed)

Reply:
i've made the change that i think you've suggested, but it might still not be quite right. Let me know if I should do some more work on it

Reply:
Ummm.... shouldn't it be a `belongs_to` on the `ContentAsset` model. Then a `has_one` or `has_many` on the `ContentPage` model.

Reply:
Yes, sorry, follow Rob's lead - I got it backwards.  ContentAsset belongs to the ContentPage and ContentPage has_many many ContentAssets.